terraform/application/config/production.tfvars.json

```diff
@@ -18,5 +18,9 @@
     "start_minute": 0
   },
   "external_url": "https://register-national-professional-qualifications.education.gov.uk/healthcheck",
-  "enable_logit": true
+  "enable_logit": true,
+  "deploy_redis_cache": true,
+  "redis_cache_family": "P",
+  "redis_cache_capacity": 2,
```

Comment:
maybe 1 instead? as we have 1 in ECF right now?

Reply:
Might be worth running this past Colin; I think we tend to use smaller capacity for queues (1/P/Premium) and larger capacity for cache (2/C/Standard)

Reply:
Colin suggested we use a premium with high network performance

Reply:
So should we keep (2/P/Premium) then?

![image](https://github.com/user-attachments/assets/2451458e-89f7-41a4-966c-b009a3f7c30c)

Reply:
yes you're right, let's keep it